tests/core/files/overdrive/overdrive_availability_advantage.json

```diff
@@ -14,8 +20,8 @@
     ],
     "availabilityType": "Normal",
     "available": true,
-    "copiesAvailable": 3,
-    "copiesOwned": 3,
+    "copiesAvailable": 4,
+    "copiesOwned": 4,
```

Comment:
The aggregate "copiesAvailable" and "copiesOwned" values here will make sense only in the context OD Advantage library ID 61. They're not the same values that would come back if the context were the main OD library or Advantage account 63.  It might make sense to rename this file to acknowledge that.

Reply:
There's mistake here:  the number should be 11.  I will verify if the aggregate number is different in the child account context.